cardano-cli/src/Cardano/CLI/Types/Common.hs

```diff
@@ -122,6 +123,10 @@ newtype ProposalUrl = ProposalUrl
   { unProposalUrl :: L.Url
   } deriving (Eq, Show)
 
+newtype ProposalBinary = ProposalBinary
+  { unProposalBinary :: Text
```

Comment:
Why binary is a text? I think I'm missing some context here.

Reply:
Right, the type we actually assume when reading is `Bytestring`: https://github.com/IntersectMBO/cardano-cli/blob/8def78a0ceca29027e7fbabcc9e5480b416eb231/cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/Hash.hs#L49

Reply:
@carbolymer> Changed to have the `newtype` wrap a `ByteString` :+1: